ios/browser/application_context/brave_application_context_impl.mm

```diff
@@ -66,6 +70,17 @@
   return local_data_files_service_.get();
 }
 
+ai_chat::LeoLocalModelsUpdater*
+BraveApplicationContextImpl::leo_local_models_updater() {
+  if (!leo_local_models_updater_) {
```

Comment:
I assume this should return `nullptr` if the feature is not enabled. Right now the updater is always created for no reason.

Reply:
This went away in 7e443e164c9e4952e57e1d040e9f300d7fa521e2